src/libs/actions/IOU.ts

```diff
@@ -4161,11 +4192,13 @@ function categorizeTrackedExpense(trackedExpenseParams: TrackedExpenseParams) {
     optimisticData?.push(...moveTransactionOptimisticData);
     successData?.push(...moveTransactionSuccessData);
     failureData?.push(...moveTransactionFailureData);
-    const parameters = {
-        ...reportInformation,
+
+    const {linkedTrackedExpenseReportAction: ignore, ...reportInformationForAPI} = reportInformation;
+
+    const parameters: CategorizeTrackedExpenseApiParams = {
+        ...reportInformationForAPI,
```

Comment:
NAB: Can we go back to how it was before with explicitly setting linkedTrackedExpenseReportAction undefined? I find it to be a little more clear. Please let me know why if we can't; I'm curious.

Reply:
I like that a little better. Thanks for explaining the issue.